lib/Sema/CSDiag.cpp

```diff
@@ -3487,9 +3487,30 @@ typeCheckArgumentChildIndependently(Expr *argExpr, Type argType,
 /// Diagnose an argument labeling issue, returning true if we successfully
 /// diagnosed the issue.
 static bool diagnoseArgumentLabelError(Expr *expr,
```

Comment:
Can you add a comment with an example of when `fnExpr` can be null? `dyn_cast_or_null` is pretty unusual.